substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/TimeSubstitutions.java

```diff
@@ -0,0 +1,23 @@
+package com.oracle.svm.core.jdk;
```

Comment:
> LGTM but there are some styling issues and it's missing the copyright header.

@galderz please take care of the styling issues. Ensure that `mx eclipseformat` and `mx checkstyle` pass without issues.